test/timezone/lookup/test_geonames.rb

```diff
@@ -19,11 +19,11 @@ def etc_data
     }
   end
 
-  def lookup(body = nil, &_block)
+  def lookup(body = nil, &block)
     config = OpenStruct.new
     config.username = 'timezone'
     config.request_handler = HTTPTestClientFactory.new(body)
-    yield config if block_given?
+    yield config if block
```

Comment:
To keep consistency with the code should we use `block_given?` here?

Reply:
This is a Rubocop fix - https://msp-greg.github.io/rubocop-performance/RuboCop/Cop/Performance/BlockGivenWithExplicitBlock.html

I'm not sure why the previous version would be preferable.